api/MutationRecord.json

```diff
@@ -303,209 +303,209 @@
             "standard_track": true,
             "deprecated": false
           }
-        },
-        "nextSibling": {
-          "__compat": {
-            "mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationRecord/nextSibling",
-            "support": {
-              "chrome": {
-                "version_added": null
-              },
-              "chrome_android": {
-                "version_added": null
-              },
-              "edge": {
-                "version_added": null
-              },
-              "edge_mobile": {
-                "version_added": null
-              },
-              "firefox": {
-                "version_added": null
-              },
-              "firefox_android": {
-                "version_added": null
-              },
-              "ie": {
-                "version_added": null
-              },
-              "opera": {
-                "version_added": null
-              },
-              "opera_android": {
-                "version_added": null
-              },
-              "safari": {
-                "version_added": null
-              },
-              "safari_ios": {
-                "version_added": null
-              },
-              "samsunginternet_android": {
-                "version_added": null
-              },
-              "webview_android": {
-                "version_added": null
-              }
-            },
-            "status": {
-              "experimental": false,
-              "standard_track": true,
-              "deprecated": false
+        }
+      },
+      "nextSibling": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationRecord/nextSibling",
+          "support": {
+            "chrome": {
+              "version_added": null
```

Comment:
This should be `true` for `chrome`, `chrome_android`, and `webview_android`. You can check which members are present in the link. https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/dom/mutation_record.idl?q=MutationRecord+file:%5Esrc/third_party/blink/renderer/+package:%5Echromium$&dr=C&l=31

Reply:
Actually, I was thinking about saving the changes to the data itself for another PR (I found out about this during the Chromium API checks), as to keep this PR small and quick, but I do plan to add these values on once it has been merged!  (Alternatively, I’m happy to make the changes if anyone thinks it’s better to do it in one go.)

Reply:
That's fine. At least correct this item.

Reply:
Thinking about it a little more, we might as well just fix it all in one go!